app/create_file.py

```diff
@@ -1 +1,49 @@
-# write your code here
+from datetime import datetime
+import os
+import sys
+
+
+def create_file_from_terminal() -> None:
+    commands = sys.argv
+    if "-f" in commands:
+        index_f = commands.index("-f")
+        file_name = commands[index_f + 1:]
+
+        if "-d" in commands:
+            index_d = commands.index("-d")
+            name_dir = commands[index_d + 1:index_f]
+            dir_path = os.path.join(*name_dir)
+
+            if not os.path.exists(dir_path):
+                os.makedirs(dir_path)
```

Comment:
This `if` check can be replaced by adding the `exist_ok=True` parameter, which prevents an error if the directory already exists. This is a more Pythonic approach.
```suggestion
            os.makedirs(dir_path, exist_ok=True)
```